autoware_map_msgs/msg/PointCloudMapWithID.msg

```diff
@@ -0,0 +1,3 @@
+# Pointcloud data with ID
+string id
+sensor_msgs/PointCloud2 pointcloud
```

Comment:
How about adding
```yaml
geometry_msgs/Point cell_center
geometry_msgs/Vector3 cell_size
```
Also what do you think about adding a center and [geometry_msgs/Vector3](https://github.com/ros2/common_interfaces/blob/rolling/geometry_msgs/msg/Vector3.msg) size information?

This way, the client can reason about this map cell more independently.

And you could name it `CloudMapCellWithId.msg`

Reply:
Agreed with the `CellWithID` proposal! https://github.com/autowarefoundation/autoware_msgs/pull/39/commits/bf074437af0e621f22333d334d1e8b17315c1796

Reply:
How about this? I think it is better to use this style instead of having center & size, since we can hold a strict boundary information.
```
geometry_msgs/Point min_point
geometry_msgs/Point max_point
```
https://github.com/autowarefoundation/autoware_msgs/pull/39/commits/b5ef05fb21fd4350c132e648707978046de22139

Reply:
Sorry didn't see this but could you answer [here](https://github.com/autowarefoundation/autoware_msgs/pull/39#discussion_r991915560)?